api/src/schemas.ts

```diff
@@ -293,6 +293,21 @@ export const schemas = {
       })
     }
   },
+  updateMyClassroomMode: {
+    body: Type.Object({
+      isClassroomAccount: Type.Boolean()
+    }),
+    response: {
+      200: Type.Object({
+        message: Type.Literal('flash.classroom-mode-updated'),
+        type: Type.Literal('success')
+      }),
+      400: Type.Object({
```

Comment:
In the route, this is defined as a `500`.

Reply:
In the old api, this is defined as a `403`.

Reply:
I guess we want everything to be the same then?